common/browser.go

```diff
@@ -588,14 +587,7 @@ func (b *Browser) NewContext(opts sobek.Value) (*BrowserContext, error) {
 		return nil, err
 	}
 
-	browserCtxOpts := NewBrowserContextOptions()
-	if err := browserCtxOpts.Parse(b.ctx, opts); err != nil {
-		err := fmt.Errorf("parsing newContext options: %w", err)
-		spanRecordError(span, err)
```

Comment:
It seems like the call to `spanRecordError` is lost in the new code. Is this intentional?

Reply:
Good catch :) That's one of the issues with de-sobek-ification. Since the option parsing now happens in the mapping layer, but the trace span starts in [the business logic](https://github.com/grafana/xk6-browser/blob/3f26d5cd37bac7a306a99e3e730ee093a7e8f12f/common/browser.go#L571-L573), it's not viable to add this error to the span. I've created #1393 to discuss what we can do for these cases.